src/App.js

```diff
@@ -1,15 +1,32 @@
-import React from 'react';
+import React from 'react'
+const todoList = [
+  {
+    id: 1,
+    title: "Complete assignment list"
+  },
+  {
+    id: 2,
+    title: "Complete assignment map"
+  },
+  {
+    id: 3,
+    title: "Complete assignment objects"
+  }
+];
 
 function App() {
+  const listItems = todoList.map((number) =>
+    <li key={number["id"].toString()}>
+      {number["title"]}
+    </li >
+  );
```

Comment:
Very cool that you stored the values in a variable before the return statement! 💯 

Just a couple of things you should consider in the future:

- Try to use names that make sense while mapping. In this case `item` makes more sense than `number`.
- There is no need to make the id a string. You can use numbers or strings in the key as long it has a unique value.